types/User.d.ts

```diff
@@ -0,0 +1,6 @@
+export default interface User {
+  image: string,
+  name: string,
+  description: string,
+  links: string[],
```

Comment:
Per i link definirei un'interfaccia `SocialLink` composta da un `type` che potrebbe essere a sua volta un enum (i.e.: `SocialTypes`) e `url` in modo da permetterci di applicarvi logica come colori, icone, ecc.

L'interfaccia potrebbe benissimo essere utile ovunque per qualsiasi tipo di social link, dai principali della community ad eventuali link presenti in un evento piuttosto che i nostri.

Reply:
Si ci può stare, l'avevo tenuta basic perché non sapevo se/quanti link avessimo intenzioni di aggiungere 👍